Hunting Queries/OfficeActivity/NewBotAdded.yaml

```diff
@@ -0,0 +1,35 @@
+id: bf76e508-9282-4cf1-9cc1-5c20c3dea2ee
+name: Previously unseen bot or applicaiton added to Teams
+description: |
+  'This hunting query helps identify new, and potentially unapproved applications or bots being added to Teams.'
+requiredDataConnectors:
+  - connectorId: Office365
+    dataTypes:
+      - OfficeActivity (Teams)
+tactics:
+  - Persistence
+  - Collection
+relevantTechniques:
+  - T1176
+  - T1119
+query: |
+
+  // If you have more than 14 days worth of Teams data change this value
+  let data_date = 14d;
+  let historical_bots = (
+  OfficeActivity
+  | where TimeGenerated > ago(data_date)
+  | where OfficeWorkload =~ "MicrosoftTeams"
+  | where isnotempty(AddonName)
+  | project AddonName);
```

Comment:
project can be changed to distinct to have unique list to use with in operator later.

Reply:
Yeah, not making any substantive changes unless required by DataType change.  Will leave for now and we can change later if needed.